src/tanxconnector/client.py

```diff
@@ -444,7 +445,7 @@ def execute_internal_transfers(self, body: InternalTransferProcessBody):
         r = self.session.post('/sapi/v1/internal_transfers/v2/process/', json=body)
         return r.json()
 
-    def intiate_and_process_internal_transfers(self, key_pair: dict, organization_key: str, api_key: str, currency: str, amount: float,
```

Comment:
Can you add support for both functions?
If we simply remove this, it'll be a breaking change.
Add a logger with a warning log for the old version, that this function will be removed in future versions

Reply:
as its an internal call we can make this change, also, updated the readme for the same call (new commit
)